### ISSUE TITLE
filter with XXX_contains="" never returns results

### ISSUE THREAD

**Issue by pso-aeb** [AEB-labs/cruddl#28]
```graphql
query {
    allOrders(filter: {orderNumber_contains: ""}) {
        id
        orderNumber
    }
}
```

A query like this always returns an empty list (if used with an ArangoDB-backend).

**Comment by henkesn** [comment]
Fixed in 0.5.1

**Comment by Yogu** [comment]
reported a bug in arangodb/arangodb#5640

btw, we should escape characters that are special characters for LIKE here.
